connaisseur/alert.py

```diff
@@ -151,11 +151,11 @@ def __get_headers(receiver_config):
 def send_alerts(
     admission_request: AdmissionRequest, admit_event: bool, reason: str = None
 ):
+    if not __is_not_hook_image(admission_request):
```

Comment:
This is a double negative and since `__is_not_hook_image` is not used elsewhere (as far as i can see), you should change that to `__is_hook_image`